cadctap/cadctap/tests/data/help_create.txt

```diff
@@ -12,6 +11,8 @@ positional arguments:
   TABLENAME             name of the table (<schema.table>) in the tap service
 
 optional arguments:
+  -a                    login as anonymous user, short form
```

Comment:
login as anonymous? I would rather say use the service anonymously.

Reply:
Changed.